ide-support/lsp/io.helidon.lsp.server/src/main/java/io/helidon/lsp/server/management/MavenSupport.java

```diff
@@ -0,0 +1,153 @@
+package io.helidon.lsp.server.management;
+
+import io.helidon.lsp.server.utils.ShellCommandHelper;
+import org.apache.maven.shared.invoker.DefaultInvocationRequest;
```

Comment:
Take a look at the common/maven module and the class called `MavenCommand`, you can use it instead of the maven invoker, it is standalone (no external dependencies).